codex-rs/core/src/is_safe_command.rs

```diff
@@ -75,7 +75,12 @@ fn is_safe_to_call_with_exec(command: &[String]) -> bool {
 fn try_parse_bash(bash_lc_arg: &str) -> Option<Tree> {
     let lang = BASH.into();
     let mut parser = Parser::new();
-    parser.set_language(&lang).expect("load bash grammar");
+
+    // If the language fails to load we consider the input unparseable, so we
```

Comment:
If this is happens, this is a logical error where our treesitter dependency is not set up correctly, so we should not swallow this error.